sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/Schema.java

```diff
@@ -222,6 +224,9 @@ public boolean isDateType() {
     public boolean isContainerType() {
```

Comment:
<!--thread_id:cc_180488663_t; commit:7bf9d2048e8d568ee9280629db391775cc490358; resolved:0-->
<!--section:context-quote-->
> **XuMingmin** wrote:
> while, my thought is MAP-> MAP, ARRAY -> CONTAINER, ROW -> COMPOSITE, to make it clear for the backend types.

<!--section:body-->
I think it's actually just another type of container. i.e. one way of imagining .a map is it's just a container of key-value pairs.